pom.xml

```diff
@@ -631,7 +631,7 @@
         <carbon.kernel.registry.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version.range>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.17.86</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.18.52-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
need to upgrade the version with the released version after merging: https://github.com/wso2/carbon-identity-framework/pull/3055